CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 The `Type=notify` directive is now supported when running VAST under
+  systemd supervision.
```

Comment:
I wouldn't know how to interpret this changelog entry. Can it be written so that non-systemd experts understand it as well? What is a directive? Where does the assignment come from?

Reply:
This became clearer when I read the systemd config, by the way.

Reply:
I've reworded this a bit to make it clearer this is talking about systemd-related changes.